dataset.go

```diff
@@ -21,6 +21,8 @@ func LoadDataset(language LangType) (ds Dataset, _ error) {
 		raw = GoIngredients
 	case SwiftLang:
 		raw = SwiftIngredients
+	case KotlinLang:
```

Comment:
Can you rebase and/or remove this change?

Reply:
I rebased branch, but that didn't do anything